### ISSUE TITLE
[Metis] 5.1.0 test fail on El Capitan

### ISSUE THREAD

**Issue by davydden** [Homebrew/homebrew-science#3438]
`brew test metis` fails with
```
/usr/local/Cellar/metis/5.1.0/bin/gpmetis
/usr/local/Cellar/metis/5.1.0/share/metis/graphs/4elt.graph
2

file: /usr/local/Cellar/metis/5.1.0/share/metis/graphs/4elt.graph.part.2, mode: w, [WritePartition]: Operation not permitted
Failed on gk_fopen()
******************************************************************************
METIS 5.0 Copyright 1998-13, Regents of the University of Minnesota
 (HEAD: , Built on: Oct 13 2015, 16:53:56)
 size of idx_t: 32bits, real_t: 32bits, idx_t *: 64bits

Graph Information -----------------------------------------------------------
 Name: /usr/local/Cellar/metis/5.1.0/share/metis/graphs/4elt.graph, #Vertices: 15606, #Edges: 45878, #Parts: 2

Options ---------------------------------------------------------------------
 ptype=kway, objtype=cut, ctype=shem, rtype=greedy, iptype=metisrb
 dbglvl=0, ufactor=1.030, no2hop=NO, minconn=NO, contig=NO, nooutput=NO
 seed=-1, niter=10, ncuts=1

Direct k-way Partitioning ---------------------------------------------------
```
Anybody see the same?

Execution of the same test from the command line works fine
```
$ gpmetis /usr/local/Cellar/metis/5.1.0/share/metis/graphs/4elt.graph 2
******************************************************************************
METIS 5.0 Copyright 1998-13, Regents of the University of Minnesota
 (HEAD: , Built on: Oct 13 2015, 16:53:56)
 size of idx_t: 32bits, real_t: 32bits, idx_t *: 64bits

Graph Information -----------------------------------------------------------
 Name: /usr/local/Cellar/metis/5.1.0/share/metis/graphs/4elt.graph, #Vertices: 15606, #Edges: 45878, #Parts: 2

Options ---------------------------------------------------------------------
 ptype=kway, objtype=cut, ctype=shem, rtype=greedy, iptype=metisrb
 dbglvl=0, ufactor=1.030, no2hop=NO, minconn=NO, contig=NO, nooutput=NO
 seed=-1, niter=10, ncuts=1

Direct k-way Partitioning ---------------------------------------------------
 - Edgecut: 146, communication volume: 147.

 - Balance:
     constraint #0:  1.000 out of 0.000

 - Most overweight partition:
     pid: 0, actual: 7805, desired: 7803, ratio: 1.00.

 - Subdomain connectivity: max: 1, min: 1, avg: 1.00

 - Each partition is contiguous.

Timing Information ----------------------------------------------------------
  I/O:          		   0.010 sec
  Partitioning: 		   0.006 sec   (METIS time)
  Reporting:    		   0.001 sec

Memory Information ----------------------------------------------------------
  Max memory used:		   1.508 MB
******************************************************************************
```

Perhaps the input file has to be moved to a temp folder?

**Comment by davydden** [comment]
this worked for me
```
  test do
    cp_r share, testpath
    ["4elt", "copter2", "mdual"].each do |g|
      system "#{bin}/graphchk", "#{testpath}/share/metis/graphs/#{g}.graph"
      system "#{bin}/gpmetis", "#{testpath}/share/metis/graphs/#{g}.graph", "2"
      system "#{bin}/ndmetis", "#{testpath}/share/metis/graphs/#{g}.graph"
    end
    system "#{bin}/gpmetis", "#{testpath}/share/metis/graphs/test.mgraph", "2"
    system "#{bin}/mpmetis", "#{testpath}/share/metis/graphs/metis.mesh", "2"
  end
```

**Comment by dpo** [comment]
Will be fixed by #3483.
